helm/docs/upgrading.md

```diff
@@ -98,6 +99,38 @@ A minimal example `values.yaml` may be found
    fiftyone migrate --info
    ```
 
+#### FiftyOne Enterprise v2.8+ `initContainer` Changes
+
+FiftyOne Enterprise v2.8.0 introduces numerous changes to the default settings
+for each system's `initContainers`.
+
+1. `initContainers` will get, by default, the container security context
+   shown below in order to comply with kubernetes security best practices.
+   This configuration prevents privilege escalation and running any
+   initialization processes as root.
+
+    ```yaml
+      containerSecurityContext:
+        allowPrivilegeEscalation: false  # Disables privilege escalation
+        runAsNonRoot: true  # Disables running as the `root` user
+        runAsUser: 1000  # Runs the init processes as UID 1000
+    ```
+
+1. `initContainers` will get, be default, the resources shown below.
+   These initialization processes are designed to be small and lightweight
+   and can therefore set small resource requests and limits instead of using
+   a cluster's defaults.
```

Comment:
```suggestion
1. `initContainers` default to the resources shown below.
   These initialization processes are lightweight
   and can therefore set small resource requests and limits instead of using
   a cluster's defaults.
```